pkg/apis/jenkins/v1alpha2/jenkins_types.go

```diff
@@ -310,6 +310,9 @@ type JenkinsMaster struct {
 	// Plugins contains plugins required by user
 	// +optional
 	Plugins []Plugin `json:"plugins,omitempty"`
+
+	// EnableCSRFProtection allows you to toggle CSRF Protection on Jenkins
+	EnableCSRFProtection bool `json:"enableCSRFProtection,omitempty"`
```

Comment:
The CSRF protection should be enabled by default to achieve this field should be:
```
// DisableCSRFProtection allows you to disable CSRF Protection on Jenkins
DisableCSRFProtection bool `json:"disableCSRFProtection"`
```

Reply:
updated